contoso_supermarket/developer/freezer_monitoring/src/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  mqtt-broker:
+    image: charrisagoracr.azurecr.io/mqtt-broker:latest
```

Comment:
does the image have to be provided here ? if yes, let's use placeholders like __acrName__ and there is a GitHub action that I use to replace all of the them with the right values

Reply:
This docker-compose.yaml file is not used. I have removed it.